deprecated.go

```diff
@@ -1,8 +1,9 @@
 package gin
 
 import (
-	"github.com/gin-gonic/gin/binding"
 	"net/http"
+
+	"github.com/Zaibon/gin/binding"
```

Comment:
GoImports in Sublimetext automatically separes the external packages, it looks like a good practice in Go. https://packagecontrol.io/packages/GoImports

Also, check out the sample code: https://golang.org/doc/code.html#Library